Extensions/pokes.js

```diff
@@ -135,11 +135,13 @@ XKit.extensions.pokes = {
 			}
 
 			if (XKit.extensions.pokes.preferences.allow_fullwidth.value) {
-				poke_html = '<div class="poke ' + poke_class + shiny_class + '" data-pokeid="' + db_nr + '" data-pokesortid="' + poke_sortid + '" data-pokename="' + poke_name + '" data-pokegender="' + poke_gender + '" style="left:' + xpos + 'px; margin-top:' + ypos + 'px;">' +
-				'<img src="' + poke_sprite + '" alt="' + poke_name + '"/>' + '</div>';
+				poke_html = `<div class="poke ${poke_class}${shiny_class}" data-pokeid="${db_nr}" data-pokesortid="${poke_sortid}" data-pokename="${poke_name}" data-pokegender="${poke_gender}" style="left:${xpos}px; margin-top:${ypos}px;">` +
+				`<img src="${poke_sprite}" alt="${poke_name}"/>'` + 
```

Comment:
`img` element should be indented one, also you forgot to remove the single quote at the end here when converting to template